src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function DropDown(props) {
+    const { value, data, styleClass, onChange } = props;
+    const handleChange = (event) => {
+        console.log(event.target.value);
+    };
+    return (
+        <div className={`mb-3 xl:w-96 ${styleClass}`}>
+            <select value={value} className='' onChange={handleChange}>
```

Comment:
- It will be good to remove `className` attr because it has an empty value.

Reply:
I will use it in future  to change the style.